docker-compose-prod.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.2'
+
+services:
+  redis:
+    image: redis:7-alpine
+
+  server: &server-config
```

Comment:
We can define a base config for the server and then overwrite the nested values as required.
Sample: https://github.com/the-deep/deeper/blob/main/docker-compose.yml